pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -84,15 +90,67 @@ async def run(self):
         if self.assembly_type is AssemblyTypes.STREAM:
             await self._rebase_and_build_for_stream()
         else:
+            # Check if microshift advisory is defined in assembly
+            if 'microshift' not in advisories:
+                self.advisory_num = await self.create_microshift_advisory()
```

Comment:
So as I understand this will create the advisory always when this job is run? I was hoping we had a separate flag to do this

Reply:
if this run for a named release and microshfit is not created then it will create it, I think flag don't have much value